urllib3/util/ssl_.py

```diff
@@ -195,6 +197,16 @@ def create_urllib3_context(ssl_version=None, cert_reqs=ssl.CERT_REQUIRED,
     """
     context = SSLContext(ssl_version or ssl.PROTOCOL_SSLv23)
 
+    if hasattr(context, 'urllib3_wrapper_context'):
```

Comment:
Actually, this is immune from test coverage, and I don't believe there's any real value obtained in including a test for it, so I simply removed it.